modules/015-admission-policy-engine/crds/doc-ru-operation-policy.yaml

```
@@ -18,6 +18,25 @@ spec:
                     - `Warn` — аналогично `Dryrun`, но дополнительно к информации о событии будет выведена информация о том, из-за какого ограничения (constraint) был бы запрет действия, если бы вместо `Warn` использовался `Deny`.
                 policies:
                   properties:
+                    pods:
+                      description: |
+                        Настройки для Pod-политик.
+                      properties:
+                        denyTolerations:
+                          description: |
+                            Запрещает использование толераций к указанным ключам таинтов. Проверяются только ключи (`spec.tolerations[*].key`).
+                          properties:
+                            enabled:
+                              description: "Включает проверку. По умолчанию: false."
```

```suggestion
                              description: "Включает проверку."
```

Значение по умолчанию отобразится в доке, т.к. оно указано в англ. версии.